app/Services/MagazineService.php

```diff
@@ -28,9 +28,9 @@ public function getMagazineByShortName($shortName)
      *
      * @return \Illuminate\Database\Eloquent\Collection
      */
-    public function getAllMagazines()
+    public function getAllMagazines($order = 'Asc')
     {
-        return $this->magazineRepo->find();
+        return $this->magazineRepo->find($order);
     }
 
     /**
```

Comment:
Hi @aabi01 sorry but not yet. Was little too busy in office. Leaving office now at 9:45 pm local time.

I'll merge tomorrow.